plugins/modules/dcnm_image_policy.py

```diff
@@ -592,7 +592,7 @@ def __init__(self, params):
             raise ValueError(msg) from error
 
         self.delete = ImagePolicyDelete()
-        self.merged = Merged(params)
+        self.replaced = Replaced(params)
```

Comment:
These diffs don't look related... probably need to merge latest `develop` changes into this branch.

Reply:
Merged develop into this branch.